Test/FMI3/parser_test_xmls/convenience/valid/get_variable_by_alias_name1/modelDescription.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <fmiModelDescription fmiVersion="3.0" modelName="" instantiationToken="">
+
 <ModelExchange modelIdentifier="myModelIdentifier" />
 
 <ModelVariables>
-  <!-- Variables with same VR -->
-  <Int32 name="v1" valueReference="1"/>
-  <Int32 name="v2" valueReference="2"/>
-  <Int32 name="v3" valueReference="1"/>
+    <Float64 name="v1" valueReference="1">
+        <Alias name="v1_a1"/>
+    </Float64>
```

Comment:
Git believes this file was moved (the one from convenience). Ignore the diff, it's all new.